iceoryx_hoofs/platform/win/include/iceoryx_hoofs/platform/platform_settings.hpp

```diff
@@ -34,7 +34,10 @@ constexpr const char IOX_PATH_SEPARATORS[] = "\\/";
 // so that the code with stub implementation at least compiles
 constexpr uint64_t IOX_UDS_SOCKET_MAX_MESSAGE_SIZE = 1024U;
 constexpr char IOX_UDS_SOCKET_PATH_PREFIX[] = "";
+constexpr const char IOX_LOCK_FILE_PATH_PREFIX[] = "C:\\Windows\\Temp\\";
```

Comment:
Normally `C:\Windows\temp` is used to store temporary data from the operating system or system processes. I think that this is not our usecase. Application-related data is normally stored under `C:\Users\user\AppData\Local\Temp`.
Now the question is how to retrieve this path correctly. One possible way is to look at the ``%TEMP%` environment variable or maybe use [GetTempPath2A](https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettemppath2a).

It's not needed to fix in this PR because temp under windows will work but it is at least an issue worth.
- [ ] fix path for temp files under windows or create an issue to follow up

Reply:
Added as task to issue #33